src/saturn/stage_02.c

```diff
@@ -98,7 +98,7 @@ INCLUDE_ASM("asm/saturn/stage_02/f_nonmat", f60DDF64, func_060DDF64);
 INCLUDE_ASM("asm/saturn/stage_02/f_nonmat", f60DE178, func_060DE178);
 
 // SAT func_060DE2B0
-void AxeKnightDeath(void) {
+void CreateExplosionPuff(void) {
```

Comment:
I am not sure if renaming this is correct or will work, so I'm updating, but can revert if required